### PR TITLE
Refactor trip to VJ

### PR DESCRIPTION
**Pull request by koch-t**

### CODE REVIEW

File: cli.c [hunk 1]
```diff
@@ -284,21 +284,21 @@ int main (int argc, char *argv[]) {
                         }
                     }
                     #endif
-                    #if RRRR_MAX_BANNED_TRIPS > 0
+                    #if RRRR_MAX_BANNED_JOURNEYS > 0
                     else
-                    if (strncmp(argv[i], "--banned-trip-offset=", 21) == 0) {
+                    if (strncmp(argv[i], "--banned-vj-offset=", 21) == 0) {
```
Comment (skinkie) [inline cli.c]: the 21 means something here it means the number of bytes to compare, you must change that too junior-developer. also at line 293.

File: cli.c [hunk 2]
```diff
@@ -426,7 +426,7 @@ int main (int argc, char *argv[]) {
         puts(result_buf);
     }
 
-    /* When searching clockwise we will board any trip that will bring us at
+    /* When searching clockwise we will board any vj that will bring us at
```
Comment (skinkie) [inline cli.c]: think vj in comments should be in full

File: cli.c [hunk 3]
```diff
@@ -70,7 +70,7 @@ static void set_add_sp (spidx_t *set,
 #endif
 
 
-#if RRRR_MAX_BANNED_TRIPS > 0
+#if RRRR_MAX_BANNED_JOURNEYS > 0
```
Comment (skinkie) [inline cli.c]: Then rename this to RRRR_MAX_BANNED_VEHICLE_JOURNEYS

File: router.c [hunk 4]
```diff
@@ -1079,7 +1079,7 @@ write_state(router_t *router, router_request_t *req,
     router->best_time[stop_index] = time;
     this_state->time              = time;
     this_state->back_journey_pattern = jpp_index;
-    this_state->back_trip         = trip_offset;
+    this_state->back_vj = vj_offset;
```
Comment (skinkie) [inline router.c]: something with spaces and alignment.

File: router.c [hunk 5]
```diff
@@ -1289,57 +1289,57 @@ void router_round(router_t *router, router_request_t *req, uint8_t round) {
...
+                fprintf(stderr, "    on board vj %d considering time %s \n",
+                                vj_index, timetext(time));
                 #endif
 
                 /* Target pruning, section 3.1 of RAPTOR paper. */
                 if ((router->best_time[router->target] != UNREACHED) &&
                     (req->arrive_by ? time < router->best_time[router->target]
                                     : time > router->best_time[router->target])) {
-                    #ifdef RRRR_TRIP
+                    #ifdef RRRR_VJ
```
Comment (skinkie) [inline router.c]: CHange the RRRR_VJ to RRRR_DEBUG_VEHICLE_JOURNEY

File: router_dump.c [hunk 6]
```diff
@@ -66,19 +66,19 @@ void dump_results(router_t *router) {
 }
 
 #if 0
-/* WARNING we are not currently storing trip IDs so this will segfault */
-void dump_trips(router_t *router) {
+/* WARNING we are not currently storing vehicle_journey IDs so this will segfault */
```
Comment (skinkie) [inline router_dump.c:69]: Is this still a true statement?

Comment (koch-t) [inline router_dump.c:69]: No it is not, but i consider re-enabling this function it out of scope of this commit.

**Pull request merged**